clickfile.py

```diff
@@ -1058,16 +1055,209 @@ def dapps():
 
 @dapps.command("report", help="Print dapps report (from .json files)")
 @click.option("-d", "--directory", default="reports", help="Directory with reports")
+@click.option("--repo", type=click.Choice(tp.get_args(RepoType)), required=True)
+@click.option("--event_name", required=True)
```

Comment:
I suggest separate logic and remove all arguments cnnected with branches form this fucntion
1) add function to proxy ci to define tag of images
2) make this function to get only tags 

I would be better because first part is already exist and here we duplicate it 
Lets discuss